ant-node/src/node.rs

```diff
@@ -69,7 +69,7 @@ const HIGHEST_SCORE: usize = 100;
 
 /// Any nodes bearing a score below this shall be considered as bad.
 /// Max is to be 100 * 100
-const MIN_ACCEPTABLE_HEALTHY_SCORE: usize = 5000;
+const MIN_ACCEPTABLE_HEALTHY_SCORE: usize = 3000;
```

Comment:
Just checking if this was an intended change? Did not see it in the PR description

Reply:
added in the PR description.
thx for remind